YoutubeExplode/ReverseEngineering/Responses/PlaylistResponse.cs

```diff
@@ -64,6 +65,11 @@ public class Video
                 .GetProperty("author")
                 .GetString();
 
+            public ChannelId GetAuthorId() => _root
```

Comment:
I would rename this to `GetChannelId` because Author seems to associate with a person, while here it's a channel.